screens/SignUp.js

```diff
@@ -14,8 +14,8 @@ const { height, width } = Dimensions.get("screen");
 import { useNavigation } from "@react-navigation/native";
 
 import { createUserWithEmailAndPassword } from "firebase/auth";
-import { auth, db } from "../firebase";
-import { collection, addDoc } from "firebase/firestore";
+import { auth, db } from "../config/firebase";
+import { doc, setDoc } from "firebase/firestore";
```

Comment:
In Firestore documents were getting stored to a newly generated doc ref id but it should have been with the userUid which was generated when the user passed the onboarding pages so used a different function to do that